docs/migration_guides/qi_migration.rst

```diff
@@ -60,10 +60,10 @@ Contents
     This guide uses the following naming standard to refer to the primitives:
 
     - *Primitives* - Any Sampler/Estimator implementation
-    - *Reference Primitives* - The Sampler and Estimator in :mod:`qiskit.primitives` --> ``from qiskit.primitives import Sampler/Estimator``
-    - *Aer Primitives* - The Sampler and Estimator in :mod:`qiskit_aer.primitives` --> ``from qiskit_aer.primitives import Sampler/Estimator``
-    - *Runtime Primitives* - The Sampler and Estimator in :mod:`qiskit_ibm_runtime` --> ``from qiskit_ibm_runtime import Sampler/Estimator``
-    - *Backend Primitives* - The BackendSampler and BackendEstimator in :mod:`qiskit.primitives` --> ``from qiskit import BackendSampler/BackendEstimator``
+    - *Reference Primitives* - The ``Sampler`` and ``Estimator`` in :mod:`qiskit.primitives` --> ``from qiskit.primitives import Sampler/Estimator``
+    - *Aer Primitives* - The ``Sampler`` and ``Estimator`` in :mod:`qiskit_aer.primitives` --> ``from qiskit_aer.primitives import Sampler/Estimator``
+    - *Qiskit Runtime Primitives* - The IBM's Qiskit Runtime primitive implementation. The ``Sampler`` and ``Estimator`` in :mod:`qiskit_ibm_runtime` --> ``from qiskit_ibm_runtime import Sampler/Estimator``
+    - *Backend Primitive* - Qiskit wrap for non-native backend primitives. The ``BackendSampler`` and ``BackendEstimator`` in :mod:`qiskit.primitives` --> ``from qiskit import BackendSampler/BackendEstimator``
```

Comment:
```suggestion
    - *Backend Primitives* - Qiskit wrappers for non-native backend primitives. The ``BackendSampler`` and ``BackendEstimator`` in :mod:`qiskit.primitives` --> ``from qiskit import BackendSampler/BackendEstimator``
```

Reply:
many wrappers?

Reply:
one per primitive, right? `BackendSampler` and `BackendEstimator`